pkg/delete/delete.go

```diff
@@ -82,7 +83,11 @@ func deleteSettingsObject(c dtclient.Client, entries []DeletePointer) []error {
 	errors := make([]error, 0)
 
 	for _, e := range entries {
-		externalID := idutils.GenerateExternalID(e.Type, e.ConfigId)
+		externalID, err := idutils.GenerateExternalID(coordinate.Coordinate{Type: e.Type, ConfigId: e.ConfigId})
+		if err != nil {
+			errors = append(errors, fmt.Errorf("unable to generate external id :%w", err))
```

Comment:
nitpicky, but customer-facing :colon placement :)
`"unable to generate external id: %w", err`